### DISCUSSION TITLE
Scheduling tasks that are a callable from a nested function but not enabled yet

### DISCUSSION THREAD

**Discussion by jcphlux** [agronholm/apscheduler#837] [Q&A]
In the [Scheduling tasks documentation](https://apscheduler.readthedocs.io/en/master/userguide.html#scheduling-tasks) it state for nested functions the below.
> If the callable you’re trying to schedule is either a lambda or a nested function, then you need to explicitly create a task beforehand, as it is not possible to create a reference (package.module:varname) to these types of callables.

I have all my jobs defined as nested functions but I don't want them to run unless the user has enabled them in the web interface. How do I create a task beforehand without a trigger as it has not been defined yet?

**Comment by agronholm** [comment]
A task is just a named pointer to a callable, plus some configuration. So creating a task doesn't make anything run unless you also add a schedule for it, or explicitly add a new job. Was the documentation unclear about that? Let me know so I can update it as necessary.
